src/components/MovieCatalog/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import ErrorBoundary from '../ErrorBoundary';
+import Filter from './Filter';
+import MovieList from './MovieList';
+import { genres, options, movies } from '../../data.js';
+
+class MovieCatalog extends React.Component {
```

Comment:
I would suggest extracting components that are responsible for data manipulation (in your case it's fetching data from static resource) into separate folder following [Container / Presentational approach](https://redux.js.org/basics/usage-with-react#presentational-and-container-components)